release-schema.json

```
@@ -98,6 +98,45 @@
             "null"
           ],
           "minLength": 1
+        },
+        "advancedElectronicSignatureRequired": {
+          "title": "Advanced electronic signature required",
+          "type": [
+            "boolean",
+            "null"
+          ],
+          "description": "Whether an advanced or qualified electronic signature or seal is required."
```

Also changed required -> needed to avoid RFC keyword.

```suggestion
          "description": "Whether an advanced or qualified electronic signature or seal is needed.",
          "type": [
            "boolean",
            "null"
          ]
```